lib/openstax/api/roar.rb

```diff
@@ -38,20 +37,19 @@ def standard_search(klass, routine, represent_with, options={})
           OSU::AccessPolicy.require_action_allowed!(:read, user, item)
         end
 
-        respond_with outputs, search_options.merge(represent_with_options)
+        respond_with outputs, { status: :ok, location: nil }.merge(represent_with_options)
       end
 
       def standard_create(model, represent_with=nil, options={})
-        create_options = { status: :created, location: nil }
-        represent_with_options = { user_options: options }.merge(represent_with: represent_with)
+        represent_with_options = { user_options: options, represent_with: represent_with }
 
         model.class.transaction do
           consume!(model, represent_with_options.dup)
           yield model if block_given?
           OSU::AccessPolicy.require_action_allowed!(:create, current_api_user, model)
 
           if model.save
-            respond_with model, create_options.merge(represent_with_options)
+            respond_with model, { status: :created, location: nil }.merge(represent_with_options)
           else
```

Comment:
I think replying with `200` is the right way to go, it'll probably ripple out to a bunch of spec changes that were testing for `:created`.   Still worth doing, but we should be prepared.